src/model/differ.js

```diff
@@ -243,6 +246,22 @@ export default class Differ {
 		return result;
 	}
 
+	/**
+	 * Checks whether some of buffered marker affects the data model or whether some element will change.
+	 *
+	 * @returns {Boolean} `true` if buffered markers or changes in elements will affect the data model.
+	 */
+	willDataChange() {
```

Comment:
I don't like the name of this function. I'd rather go with something like `hasDataChanges` or `dataChangeRegistered` or `registeredDataChanges`. IDK.

Reply:
`hasDataChanges()` sounds fine